packages/devtools_app/lib/src/charts/chart_controller.dart

```diff
@@ -208,6 +208,15 @@ class ChartController extends DisposableController
     _title = value;
   }
 
+  final ValueNotifier<TapNotifier> _tapNotifier =
+      ValueNotifier<TapNotifier>(TapNotifier.empty());
+
+  ValueNotifier<TapNotifier> get tapNotifier => _tapNotifier;
+
+  void setTapNotifier(TapNotifier tap) {
```

Comment:
rather than having a getter and a setter here, why not just make tapNotifier public?

Reply:
Yep, now that I have both.